lib/android/src/main/java/com/airbnb/android/react/maps/FusedLocationSource.java

```diff
@@ -0,0 +1,64 @@
+package com.airbnb.android.react.maps;
+
+import android.content.Context;
+import android.location.Location;
+import android.os.Looper;
+
+import com.google.android.gms.location.FusedLocationProviderClient;
+import com.google.android.gms.location.LocationCallback;
+import com.google.android.gms.location.LocationRequest;
+import com.google.android.gms.location.LocationResult;
+import com.google.android.gms.location.LocationServices;
+import com.google.android.gms.maps.LocationSource;
+import com.google.android.gms.tasks.OnSuccessListener;
+
+public class FusedLocationSource implements LocationSource {
```

Comment:
Just for the record:
This file is indented with 4 spaces while the rest of the project is indented with 2 spaces.
It's non blocking, we should create a separate pull request for automatic style checking/formatting.